user/search.go

```diff
@@ -79,6 +101,8 @@ func (opts renderData) QueryArgs() []sql.NamedArg {
 		sql.Named("search", opts.SearchStr()),
 		sql.Named("afterName", opts.After.Name),
 		sql.Named("omit", sqlutil.UUIDArray(opts.Omit)),
+		sql.Named("CMValue", opts.CMValue),
```

Comment:
In order to be consistent with naming conventions, should we change these to `cmvalue` and `cmtype`?

Reply:
Caza suggested the change in one of the comments above. Since C and M are separate words, they should be upper case. I changed all the references to keep it consistent.

Reply:
Ahh I see! That makes sense. We can keep them as it is.